oak_runtime/src/lib.rs

```diff
@@ -394,10 +394,14 @@ impl AuxServer {
     ) -> Self {
         let (termination_notificiation_sender, termination_notificiation_receiver) =
             tokio::sync::oneshot::channel::<()>();
+        let runtime_proxy = runtime.clone().proxy_for_new_node(name);
```

Comment:
The AuxServer is definitely on its own thread, and it communicates with other nodes using handles, so I would assume it qualifies as a node?

The reason I am creating a proxy here is that the thread local variable stores a proxy, not an `Arc<Runtime>`

Reply:
Are you sure it communicates with other nodes using handles? I think it should not. Also, it does not implement `Node`. 
Do any of the tests fail if you remove `proxy_for_new_node(name)`?

Reply:
This might also explain why `node 1` does not exist anymore (in `abitest`). We have two AuxServers, one for metrics and one for introspection. I think these servers are being created before any other node. I am guessing with the call to `proxy_for_new_node` here, these servers consume the first two node_ids, without actually being nodes.

Reply:
Aah, sorry you are right, I think the `AuxServer` does not need to set this.

I will remove it, thanks 😄

Reply:
Thanks :)